bibcheck/plugins/ostimove.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2016 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+""" Bibcheck plugin to move info for OSTI from tag
+    8564_w/y to tag 035__a/9
+"""
+
+from invenio.bibrecord import record_add_field
+import re
+
+PROVENANCE = 'OSTI'
+
+OSTI_ID = re.compile(r'^https?://(?:www\.)?osti\.gov/.*?\D(\d+)$')
```

Comment:
I find several osti links which do not match this regexp

e.g.
http://www.osti.gov/accomplishments/documents/fullText/ACC0169.pdf                                                                   
http://www.osti.gov/scitech/servlets/purl/5896179/                                                                                       
http://www.osti.gov/accomplishments/documents/fullText/ACC0068.pdf                                                                         
http://www.osti.gov/energycitations/product.biblio.jsp?osti_id=5971122'DOC-TYPE=ConferencePaper                                               
http://www.osti.gov/scitech/servlets/purl/1000476-RU4rvP/                                                                                         
http://www.osti.gov/scitech/servlets/purl/7250813-qyOwlS/                                                                                          
http://www.osti.gov/scitech/servlets/purl/6960980-5T3xqy/                                                                                                
http://www.osti.gov/scitech/servlets/purl/5473655-sQMxI2/                                                                                                  
http://www.osti.gov/scitech/biblio/10117897-from-psi-charmed-mesons
http://www.osti.gov/energycitations/product.biblio.jsp?osti_id=
http://www.osti.gov/energycitations/product.biblio.jsp?osti_id=
http://www.osti.gov/bridge/servlets/purl/495783-khmhVM/webviewable/495783.pdf
http://www.osti.gov/scitech/servlets/purl/789443/
http://www.osti.gov/scitech/servlets/purl/981461/
http://www.osti.gov/accomplishments/documents/fullText/ACC0067.pdf
http://www.osti.gov/bridge/purl.cover.jsp?purl=/4096074/4096074.pdf
http://www.osti.gov/scitech/servlets/purl/1133081/
http://www.osti.gov/scitech/servlets/purl/472669-2yrzjX/webviewable/
http://www.osti.gov/scitech/servlets/purl/654116-I7Z6F1/webviewable/